integration-test/spark3/src/test/java/org/apache/uniffle/test/ContinuousSelectPartitionStrategyTest.java

```diff
@@ -244,7 +244,7 @@ public int compare(String o1, String o2) {
               .get(RSS_CLIENT_BLOCK_ID_SELF_MANAGED_ENABLED);
       if (ClientType.GRPC == clientType && !blockIdSelfManagedEnabled) {
         // TODO skip validating for GRPC_NETTY, needs to mock ShuffleServerNettyHandler
-        // TODO: skip validating when blockId is managed in spark driver side.
+        // skip validating when blockId is managed in spark driver side.
```

Comment:
Maybe you can remove the `TODO` in the 246 line, too.

Reply:
No. todo in 246 is reasonable.